src/core/p5.Renderer.js

```diff
@@ -60,6 +60,46 @@ p5.Renderer = function(elt, pInst, isMainCanvas) {
 
 p5.Renderer.prototype = Object.create(p5.Element.prototype);
 
+p5.Renderer.prototype.push = function() {
+  return {
+    properties: {
+      _doStroke: this._doStroke,
+      _strokeSet: this._strokeSet,
+      _doFill: this._doFill,
+      _fillSet: this._fillSet,
+      _tint: this._tint,
+      _imageMode: this._imageMode,
+      _rectMode: this._rectMode,
+      _ellipseMode: this._ellipseMode,
+      _textFont: this._textFont,
+      _textLeading: this._textLeading,
+      _textSize: this._textSize,
+      _textStyle: this._textStyle
+    }
+  };
+};
+
+function assign(to, firstSource) {
```

Comment:
could this be _assign so we don't clutter this name in the global scope? a comment about it's expected use (here and in renderer subclasses) would also be helpful—or maybe that can be encapsulated in the function name.

Reply:
actually, this isn't in global scope, this is local to the 'p5.Renderer.js' module. in the minified library it'll probably end up renamed to `a()` ;-)

Reply:
still, a comment about its intended use, or more descriptive name, would be helpful.